### DISCUSSION TITLE
Lift

### DISCUSSION THREAD

**Discussion by Daek-wml** [Vdauphin/HeartsAndMinds#1649] [Question and answer] [answered]
Hello,

Lift hook does not work with SOG Prairie Fire Chinook. How to make it work to lift vehicles?

Thanks

**Comment by Vdauphin** [comment, marked as answer]
Hello,

I don't know, here is the function define what heli can lift: https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/def/mission.sqf#L592-L622
I think SOG has low `slingLoadMaxCargoMass`

Cheers
